sdk/eventhub/Azure.Messaging.EventHubs/src/EventHubClient.cs

```diff
@@ -130,17 +130,17 @@ public EventHubClient(string connectionString) : this(connectionString, null, nu
         /// </summary>
         ///
         /// <param name="connectionString">The connection string to use for connecting to the Event Hubs namespace; it is expected that the Event Hub path and SAS token are contained in this connection string.</param>
-        /// <param name="eventHubPath">The path of the specific Event Hub to connect the client to.</param>
+        /// <param name="eventHubName">The name of the specific Event Hub to connect the client to.</param>
         /// <param name="clientOptions">A set of options to apply when configuring the client.</param>
         ///
         /// <remarks>
         ///   If the connection string is copied from the Event Hub itself, it will contain the path to the desired Event Hub,
```

Comment:
There are a couple of places here that refer to `path`. Should those be `name`s instead.